internal/sourcecode/templates/laravel/octane/Dockerfile

```diff
@@ -0,0 +1,128 @@
+FROM alpine:edge
+
+LABEL fly_launch_runtime="laravel"
+
+RUN apk update
+
+# add useful utilities
+RUN apk add curl \
+    zip \
+    unzip \
+    ssmtp \
+    tzdata
+
+# php, with assorted extensions we likely need
+RUN apk add php8 \
+    php8-cli \
+    php8-pecl-mcrypt \
+    php8-soap \
+    php8-openssl \
+    php8-gmp \
+    php8-pdo_odbc \
+    php8-json \
+    php8-dom \
+    php8-pdo \
+    php8-zip \
+    php8-pdo_mysql \
+    php8-sqlite3 \
+    php8-pdo_pgsql \
+    php8-bcmath \
+    php8-gd \
+    php8-odbc \
+    php8-pdo_sqlite \
+    php8-gettext \
+    php8-xmlreader \
+    php8-bz2 \
+    php8-iconv \
+    php8-pdo_dblib \
+    php8-curl \
+    php8-ctype \
+    php8-phar \
+    php8-xml \
+    php8-common \
+    php8-mbstring \
+    php8-tokenizer \
+    php8-xmlwriter \
+    php8-fileinfo \
+    php8-opcache \
+    php8-simplexml \
+    php8-pecl-redis\
+    php8-sockets \
+    php8-pcntl \
+    php8-posix \
+    php8-pecl-swoole
+
+# node, for Laravel mix
+RUN apk add nodejs npm
+
+# supervisor, to support running multiple processes in a single app
+RUN apk add supervisor
```

Comment:
We should pull all the `apk add` into a single line, otherwise every package gets its own layer. Generally the fewer layers the better for Docker images.